docs/pages/setup/admin/daemon.mdx

```diff
@@ -1,33 +1,55 @@
 ---
-title: Teleport Daemon
-description: Teleport Daemon Configuration and Systemd Unit
+title: Run Teleport as a Daemon
+description: Configure Teleport to run as a daemon using systemd
 ---
 
-## Teleport daemon
+The Teleport binary is called `teleport`. When you run `teleport start`, the
+Teleport process runs in the foreground. On Linux systems in non-containerized
+environments, we recommend running the `teleport` binary as a daemon using
+systemd.
 
-The Teleport daemon is called `teleport` and it supports
-the following commands:
+## Prerequisites
 
-| Command | Description |
-| - | - |
-| start | Starts the Teleport daemon. |
-| configure | Dumps a sample configuration file in YAML format into standard output. |
-| version | Shows the Teleport version. |
-| status | Shows the status of a Teleport connection. This command is only available from inside of an active SSH session. |
-| help | Shows help options. |
+A Linux host where you will install Teleport. The host must be configured to run
```

Comment:
I'd probably just say "The host must have systemd installed" or "The host must be configured to use systemd". I think there's a risk of confusing people by mentioning "userspace programs", potentially leading them to think there's some additional configuration they need to do.